lib/raidenclient/RaidenClient.ts

```diff
@@ -125,19 +108,35 @@ class RaidenClient extends BaseClient {
       this.logger.error(`can't init raiden. raiden is disabled`);
       return;
     }
+    // set status as disconnected until we can verify the connection
+    this.setStatus(ClientStatus.Disconnected);
+    await this.verifyConnection();
+  }
+
+  /**
+   * Verify that Raiden REST service can be reached by attempting a `getAddress` call.
+   */
+  private verifyConnection = async () => {
+    this.logger.info(`trying to verify connection to raiden with uri: ${this.host}:${this.port}`);
     try {
+      if (this.reconnectionTimer) {
+        clearTimeout(this.reconnectionTimer);
+      }
       this.address = await this.getAddress();
       this.setStatus(ClientStatus.ConnectionVerified);
     } catch (err) {
-      this.logger.error('could not get raiden address', err);
+      this.logger.error(
+        `could not verify connection to raiden at ${this.host}:${this.port}, retrying in ${RaidenClient.RECONNECT_TIMER} ms`,
+      );
+      this.reconnectionTimer = setTimeout(this.verifyConnection, RaidenClient.RECONNECT_TIMER);
     }
   }
 
   public getRaidenInfo = async (): Promise<RaidenInfo> => {
     let channels: number | undefined;
     let address: string | undefined;
     let error: string | undefined;
-    const version = 'v0.3.0'; // Hardcoded for now until they expose it to their API;
+    const version = 'v0.100.2'; // Hardcoded for now until they expose it to their API;
```

Comment:
Why do we need this version?

Reply:
You're right, we don't need this - it's currently just for display purposes when querying for `xucli getinfo`. At the moment Raiden client does not expose its version via the API - I imagine they will in the future.

Reply:
Maybe lets just leave version blank until they expose it? I wouldn't want to have to keep manually updating this line.

Reply:
Good idea - done.